File: netfox/Core/NFXLoader.m
//
//  NFXLoader.m
//  netfox
//
//  Copyright © 2017 kasketis. All rights reserved.
//

#import "NFXLoader.h"

@implementation NFXLoader

+ (void)load
{
    SEL implementNetfoxSelector = NSSelectorFromString(@"implementNetfox");
    if ([NSURLSessionConfiguration respondsToSelector:implementNetfoxSelector])
    {
        [NSURLSessionConfiguration performSelector:implementNetfoxSelector];
    }
}

@end

File: netfox/Core/NFXLoader.swift
//
//  NFXLoader.swift
//  netfox_ios
//
//  Created by Stefan Haider on 20.03.20.
//  Copyright © 2020 kasketis. All rights reserved.
//

import UIKit

class NFXLoader {

    class func load() {
        let implementNetfoxSelector = NSSelectorFromString("implementNetfox")
        if URLSessionConfiguration.responds(to: implementNetfoxSelector) {
            URLSessionConfiguration.perform(implementNetfoxSelector)
        }
    }
}
